src/freva/cli/plugin.py

```diff
@@ -92,9 +92,9 @@ def __init__(
         )
         self.parser.add_argument(
             "--unique_output",
-            help="Append a freva run id to every output folder",
-            default=True,
-            type=bool,
+            choices=["True", "False"],
+            help="Append a Freva run id to the output/cache folder(s).",
```

Comment:
If you wanted to do it like that then I would be inclined to say you make the choices lowercase.

Reply:
Also, could you do me a favour and call this `--unique-output`. I think back when we transitioned I stayed with `unique_output` because I wasn't to sure about the implications. I think now I can say it won't matter, except that `unique_output` doesn't look nice.

Reply:
I will do both. I will also check whether in the cli, there is any option that involves capitalisation and make it lowercase, just in case.